addons/l10n_in/models/account.py

```diff
@@ -31,6 +32,9 @@ def name_get(self):
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
+    def init(self):
+        tools.create_index(self._cr, 'account_move_line_move_product_idx', self._table, ['move_id', 'product_id'])
```

Comment:
I don't like ending `idx` :slightly_smiling_face:  . It looks like `ID + x`

```suggestion
        tools.create_index(self._cr, 'account_move_line_move_id_product_id_index', self._table, ['move_id', 'product_id'])
```

https://github.com/odoo/odoo/blob/3ebfac79abb81a1ab5767c47fdc32f56b049b2df/odoo/modules/registry.py#L433